tests/rpc-handler.test.ts

```diff
@@ -191,4 +191,16 @@ describe("RPCHandler", () => {
       }, 10000);
     }
   });
+
+  it("Should return the first available RPC", async () => {
+    const module = await import("../types/rpc-handler");
+    const rpcHandler = new module.RPCHandler({
+      ...testConfig,
+      networkId: "1",
+      rpcTimeout: 1000,
+    });
+
+    const provider = await rpcHandler.getFirstAvailableRpcProvider();
+    expect(provider).not.toBeNull();
```

Comment:
Maybe also count the results, it should be one.

Reply:
It returns a single item not an array, what can I count?